tools/webpack/packages.js

```diff
@@ -156,5 +157,9 @@ module.exports = {
 				.concat( bundledPackagesPhpConfig )
 				.concat( vendorsCopyConfig ),
 		} ),
+		new MomentTimezoneDataPlugin( {
+			startYear: 2000,
+			endYear: 2030,
+		} ),
```

Comment:
This will require us to maintain the years every now and then. Does it perhaps make sense to make these dynamic?

```
const currentYear = new Date().getFullYear();
```

and then something like:

```suggestion
			startYear: currentYear - 20,
			endYear: currentYear + 10,
```

Reply:
What you're saying is true, but how does it relate to the timezone config? Yes, WordPress started in 2003, but you can have a post written earlier (in 1997 for example) and that's fine. You can also schedule a post for 2040, that's perfectly fine. Perhaps we can keep the end-year dynamic at least? Otherwise, we may easily forget to update it as we get closer to it, and as much as we should remove `moment.js` by then, WordPress is known to maintain backwards compatibility forever. This might mean that the `@wordpress/date` package will remain forever, albeit unused in Gutenberg, and that would mean that this config will also remain for the time being.

Reply:
I was also curious about other CPTs which may not adhere to when they were posted. For example, someone could create a historical timeline using CPTs where each event is a post and has a historical date. (Though does that even matter in the context of this webpack plugin?)